app/controllers/discourse_reactions/custom_reactions_controller.rb

```diff
@@ -77,15 +77,26 @@ def post_reactions_users
       likes = post.post_actions.where('deleted_at IS NULL AND post_action_type_id = ?', PostActionType.types[:like]) if !reaction_value || reaction_value == DiscourseReactions::Reaction.main_reaction_id
 
       if likes.present?
+        main_reaction = DiscourseReactions::Reaction.find_by(reaction_value: DiscourseReactions::Reaction.main_reaction_id, post_id: post.id)
+        count = likes.length.to_i
```

Comment:
why do we need to_i  here ? can length be something else than integer ?

Reply:
No😅 nice catch Ill change it